addon/components/o-s-s/smart/button.hbs

```diff
@@ -0,0 +1,30 @@
+{{! template-lint-disable u-template-lint/no-bare-button}}
+<div class="upf-smart-btn-container fx-row" ...attributes>
+  <button
+    type="button"
+    class={{this.computedSmartClasses}}
+    disabled={{@disabled}}
+    {{did-insert this.didInsert}}
+    {{on "click" this.onclick}}
+  >
+    {{#if this.intervalState}}
```

Comment:
I think this is used only for countdowns on buttons and I'm pretty sure they've been deprecated on the original OSS::Button (at least we don't use them anymore).

IMO there's no use of having it here. you can remove the condition in the template